SS.Integration.Adapter/Actors/Messages/StreamDisconnectedMsg.cs

```diff
@@ -17,5 +17,6 @@ namespace SS.Integration.Adapter.Actors.Messages
     internal class StreamDisconnectedMsg
     {
         public string FixtureId { get; set; }
+        public bool IsStreamConnected { get; set; }
```

Comment:
Same as in above comment, why do we need IsStreamConnected on StreamDisconnectedMsg???